tests/conftest.py

```diff
@@ -96,12 +91,34 @@ def pytest_generate_tests(metafunc):
     'example_script' fixture.
     """
 
+    mi_version = get_mi_version()
```

Comment:
This is all getting a bit absurd. We could just pass an environment variable in, and assume the latest version if it's not available.